core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3169,6 +3197,35 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
     FETCH_OPCODE_AND_DISPATCH ();
 #endif
 
+#if WASM_ENABLE_TAIL_CALL !=0
+  call_func_from_return_call:
+  {
+      uint32 *lp_base;
+      uint32 *lp;
+      int i;
+
+      lp_base = lp = wasm_runtime_malloc(cur_func->param_cell_num * sizeof(uint32));
```

Comment:
Should check return value, if NULL: wasm_set_exception(module, "allocate memory failed"); goto got_exception;